topomodelx/utils/sparse.py

```diff
@@ -0,0 +1,32 @@
+"""Utils for more efficient sparse matrix casting to torch."""
+
+import numpy as np
+import torch
+from scipy.sparse import _csc
+
+
+def from_sparse(data: _csc.csc_matrix):
+    """Convert sparse input data directly to torch sparse coo format.
+
+    Parameters
+    ----------
+    data : scipy.sparse._csc.csc_matrix
+        Input n_dimensional data.
+
+    Returns
+    -------
+    torch.sparse_coo, same shape as data
+        input data converted to tensor.
+    """
+    if not isinstance(data, _csc.csc_matrix):
+        raise ValueError(
+            f"Expected Data type sparse._csc.csc_matrix, found {type(data)}"
+        )
+
+    # cast from csc_matrix to coo format for compatibility
+    coo = data.tocoo()
+
+    v = torch.FloatTensor(coo.data)
```

Comment:
avoid single letter variable names: it's harder to understand + harder to handle in find-and-replace situations.